master/buildbot/steps/trigger.py

```diff
@@ -107,6 +108,7 @@ def hasRenderable(l):
         self.brids = []
         self.triggeredNames = None
         self.waitForFinishDeferred = None
+        self.reslist = []
```

Comment:
I think something like `_result_list` would be more readable.

Reply:
Makes sense, changed